en/advanced_config/bootloader_update.md

```diff
@@ -96,18 +97,25 @@ To update the bootloader:
    In this case you will not be able to install FMUv3 hardware.
    :::
 
-## Dronecode Probe Bootloader Update
+## Debug Probe Bootloader Update
 
-The following steps explain how you can "manually" update the bootloader using the dronecode probe:
+The following steps explain how you can "manually" update the bootloader using a [compatible Debug Probe](../debug/swd_debug.md#debug-probes-for-px4-hardware):
 
 1. Get a binary containing the bootloader (either from dev team or build it yourself).
-1. Connect the Dronecode probe to your PC via USB.
+
+1. Get a [Debug Probe](../debug/swd_debug.md#debug-probes-for-px4-hardware).
+   Connect the probe your PC via USB and setup the `gdbserver`.
+
 1. Go into the directory containing the binary and run the following command in the terminal:
 
    ```sh
    arm-none-eabi-gdb px4fmuv5_bl.elf
    ```
 
+   :::note
+   If the hardware is FMUv6X or newer use the in-tree PX4 bootloader.
+   :::
```

Comment:
I think this would be better? (please check command)

```suggestion
   ```sh
   arm-none-eabi-gdb px4fmuv6x_bl.elf
   ```

   :::note
   The naming pattern for the bootloader binary is the same for both in-tree and legacy PX4 bootloader.
   :::
```